ui/src/submissions/index.jsx

```diff
@@ -75,6 +78,11 @@ class Submissions extends Component {
               path={`${SUBMISSIONS_SEMINAR}/:id`}
               component={SeminarUpdateSubmissionPageContainer}
             />
+            <Route
+              exact
+              path={SUBMISSIONS_EXPERIMENT}
+              component={ExperimentSubmissionPageContainer}
+            />
```

Comment:
should be private https://github.com/inspirehep/inspirehep/blob/ba5fe0fdd9aa4daebad985376068487edccf995a/ui/src/common/PrivateRoute.jsx#L10